src/program.h

```diff
@@ -22,48 +22,53 @@ namespace ql
  */
 class quantum_program
 {
-    protected:
-        bool                        default_config;
-        std::string                 config_file_name;
-        std::vector<quantum_kernel> kernels;
+   protected:
```

Comment:
it looks as if the tab settings are different from rest of project (i.e. 4 spaces)

Reply:
fixed